components/unzipped/dashboard/Kanban/AssignedToList.js

```diff
@@ -0,0 +1,193 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import MenuItem from '@mui/material/MenuItem';
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
+import Divider from '@mui/material/Divider';
+import Checkbox from '@mui/material/Checkbox';
+import ListItemText from '@mui/material/ListItemText';
+import CloseIcon from '@mui/icons-material/Close';
+import DndFilterIcon from '../../../icons/dndFilterIcon';
+
+export const SpanStyled = styled.span`
+    display: inline-flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 25px;
+`;
+
+const StyledMenu = styled((props) => (
+    <Menu
+        elevation={0}
+        anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'right',
+        }}
+        transformOrigin={{
+            vertical: 'top',
+            horizontal: 'right',
+        }}
+        {...props}
+    />
+))(({ theme }) => ({
+
+    '& .MuiPaper-root': {
+        borderRadius: 6,
+        marginTop: "8px",
+        minWidth: 280,
+        maxWidth: 320,
+        maxHeight: 250,
+        color: "#737373",
+        boxShadow:
+            'rgb(255, 255, 255) 0px 0px 0px 0px, rgba(0, 0, 0, 0.05) 0px 0px 0px 1px, rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px',
+        '& .MuiMenu-list': {
+            padding: '2px 0',
+        },
+    },
+}));
+
+
+const USERS_ARR = [
+    {
+        userId: 1,
+        firstName: "Aalishan ",
+        lastName: "Jami"
```

Comment:
Looks like we are hard coding the user's here. The users that can be selected should be the owner of the project and any freelancers hired for the project. See how it works in the normal task list page.